website/docs/config/sandbox-config.mdx

```diff
@@ -113,7 +113,7 @@ When a sandbox is started, the implicit accounts defined in the configuration wi
                 }
             },
             "label": "Local Tezos Sandbox",
-            "protocol": "Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A",
+            "protocol": "PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY",
```

Comment:
We should use the Alpha protocol here so that we don't have to keep updating this.